### PR TITLE
Holy Priest: Updated divine word talent with new value 

### PR DESCRIPTION
**Pull request by emilnormann**
Added missing functionality as well.

### CODE REVIEW

File: src/analysis/retail/priest/holy/modules/talents/BottomRow/LightOfTheNaaru.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ class LightOfTheNaaru extends Analyzer {
               this.chastise.lightOfTheNaaruCooldownReduction) /
               1000,
           )}
-          s <small>Cooldown Reduction</small>
+          <small>Cooldown Reduction</small>
```
Comment (kfinch) [inline src/analysis/retail/priest/holy/modules/talents/BottomRow/LightOfTheNaaru.tsx]: The 's' is the units, like "30s Cooldown Reduction" - why remove it?

File: src/analysis/retail/priest/holy/modules/talents/BottomRow/DivineWord.tsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ const DAMAGE_INCREASE_FROM_CHASTISE = 0.5;
 const HOLY_FIRE_APPLICATION_DURATION = 7;
 const HEALING_INCREASE_FROM_SERENITY = 0.3;
 const CRIT_INCREASE_FROM_SERENITY = 0.2;
+const ACTIVATOR_SPELL_INCREASE = 0.3;
```
Comment (kfinch) [inline src/analysis/retail/priest/holy/modules/talents/BottomRow/DivineWord.tsx:25]: The tooltip for Divine Word on wowhead currently shows 50% - is 30% the most up to date value?

File: src/analysis/retail/priest/holy/modules/talents/BottomRow/DivineWord.tsx [hunk 3]
```diff
@@ -161,6 +163,21 @@ class DivineWord extends Analyzer {
     }
   }
 
+  onDamage(event: DamageEvent) {
+    const spellId = event.ability.guid;
+    if (
+      spellId === TALENTS.HOLY_WORD_CHASTISE_TALENT.id &&
+      this.selectedCombatant.hasBuff(TALENTS.DIVINE_WORD_TALENT.id)
+    ) {
+      this.chastiseDamage += calculateEffectiveDamage(event, DAMAGE_INCREASE_FROM_CHASTISE);
```
Comment (kfinch) [inline src/analysis/retail/priest/holy/modules/talents/BottomRow/DivineWord.tsx]: Shouldn't this be `ACTIVATOR_SPELL_INCREASE`?

**Review by kfinch — CHANGES_REQUESTED** [review]
I think I'm still not onboard with surfacing overhealing so prominently - I don't get the purpose of that for a heal like this.

File: src/analysis/retail/priest/holy/modules/talents/BottomRow/DivineWord.tsx (hunk 2 above)
Comment (emilnormann) [reply src/analysis/retail/priest/holy/modules/talents/BottomRow/DivineWord.tsx:25]: It is currently 30% on live.

**Comment by emilnormann** [comment]
> I think I'm still not onboard with surfacing overhealing so prominently - I don't get the purpose of that for a heal like this.

I feel like showing overheal does not make it harder to read and is very useful for some talents imo. And by default I would show it for all talents for consistency. For this talent specifically it also shows very well if you made good use of the sanctify healing.

**Review by kfinch — APPROVED** [review]

**Pull request merged**